backends/imgui_impl_vulkan.cpp

```diff
@@ -969,11 +970,39 @@ bool ImGui_ImplVulkan_CreateDeviceObjects()
         check_vk_result(err);
     }
 
+    if (v->DescriptorPool == VK_NULL_HANDLE)
+    {
+        // This is taken from the examples, but probably way oversized.
+        // We should try to size them based on what we actually need.
+        VkDescriptorPoolSize pool_sizes[] = {
```

Comment:
This pool setup is bad practice. While it may work on some implementations, it may fail on others. Pool sizes should be kept as close to actual usage as possible, or at least limited to the max. values an implementation offers.